tests/components/fronius/test_config_flow.py

```diff
@@ -0,0 +1,227 @@
+"""Test the fronius config flow."""
+from unittest.mock import patch
+
+from pyfronius import FroniusError
+
+from homeassistant import config_entries, setup
+from homeassistant.components.fronius.const import DOMAIN
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from tests.common import MockConfigEntry
+
+INVERTER_INFO_RETURN_VALUE = {"inverters": [{"unique_id": {"value": "1234567"}}]}
+LOGGER_INFO_RETURN_VALUE = {"unique_identifier": {"value": "123.4567"}}
+
+
+async def test_form_with_logger(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == RESULT_TYPE_FORM
+    assert result["errors"] is None
+
+    with patch(
+        "pyfronius.Fronius.current_logger_info",
```

Comment:
Are we patching in the right place?

https://docs.python.org/3/library/unittest.mock.html#where-to-patch

Reply:
Hm you are right, this is imported via `from`... I wonder why these tests worked despite this 🤨